examples/ts-vue3/src/App.vue

```diff
@@ -0,0 +1,158 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import VueHcaptcha from '@hcaptcha/vue3-hcaptcha';
```

Comment:
just for the consistency, let's use one type of quotes here: singular or double

Reply:
Fixed https://github.com/hCaptcha/vue-hcaptcha/pull/86/commits/b68905105c83328d4f25a18c2741a3eb7a0891d5